legacy/src/app/partials/nodes-list.html

```diff
@@ -1341,6 +1342,14 @@ <h4 class="page-header__dropdown-title">Add rack controller</h4>
             >
               <div class="u-truncate">{$ controller.node_type_display $}</div>
             </td>
+            <td class="p-double-row" aria-label="# of VLANs">
+              <div class="p-double-row__rows-container">
+                <div class="p-double-row__main-row"></div>
```

Comment:
Is the number of VLANs going to come in a follow up branch?

Reply:
Yes, sorry I should I mentioned that's not available yet. I have an issue ready and waiting: https://github.com/canonical-web-and-design/maas-ui/issues/2589.